src/main/resources/templates/add_sighting.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head lang="en"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="https://www.thymeleaf.org">
+    <link th:href="@{/css/addSighting.css}" rel="stylesheet" />
+    <link th:href="@{/css/application.css}" rel="stylesheet" />
+    </head>
+    <body>
+        <div class="content-container">
+            <h1>Add a whale sighting</h1>
+            <p>Please add the details of your sighting below.dfrefgerg</p>
+            <form id="add-sighting" action="" method="">
+                <label for="date">
+                    Date of Sighting
+                </label>
+                <input type="date" required/>
+                <label for="latitude">
+                    Latitude
+                </label>
+                <input type="number" required/>
+                <label for="longitude">
+                    Longitude
+                </label>
+                <input type="number" required/>
+                <input type="hidden"/>
+                <input type="submit"/>
+            </form>
+        </div>
+    </body>
+</html>
```

Comment:
Blank line at end of file
```suggestion
</html>
```